packages/manager/apps/web-domains/src/alldoms/hooks/data/useGetAllDoms.tsx

```diff
@@ -1,51 +1,27 @@
 import { useQueries } from '@tanstack/react-query';
-import {
-  getAllDomResource,
-  getServiceInformation,
-} from '@/alldoms/data/api/web-domains';
-import { TServiceDetail, TServiceProperty } from '@/alldoms/types';
-import { findContact } from '@/alldoms/utils/utils';
-import {
-  ServiceInfoContactEnum,
-  ServiceRoutes,
-} from '@/alldoms/enum/service.enum';
+import { getServiceInformation } from '@/alldoms/data/api/web-domains';
+import { ServiceRoutes } from '@/alldoms/enum/service.enum';
 
 interface UseGetDatagridServiceInfoListProps {
-  readonly allDomList: TServiceProperty[];
+  readonly names: string[];
 }
 
 export const useGetAllDoms = ({
-  allDomList = [],
+  names = [],
```

Comment:
```suggestion
  names,
```
I think you do not need to init names since you verify if it is null or undefined to do things on line 13